admin/modules/user/users.php

```diff
@@ -3587,7 +3587,7 @@ function build_users_view($view)
 			}
 			if(!$user['avatar'])
 			{
-				$user['avatar'] = "../".$mybb->settings['useravatar'];
+				$user['avatar'] = "../".str_replace('{theme}', 'images', $mybb->settings['useravatar']);
```

Comment:
Have you created an issue for not using `format_avatar()` here (and the one above) or could this be fixed with #2363?